Utilities/DataLoading/EdFi.XmlLookup.Console/Program.cs

```diff
@@ -55,32 +46,45 @@ private static void Main(string[] args)
             {
                 try
                 {
-                    using (var container = new Container())
-                    {
-                        LogConfiguration(p.Object);
+                    var container = RegisterContainer();
+                    using var scope = container.BeginLifetimeScope();
 
-                        // configure DI container
-                        ConfigureCompositionRoot(container, p.Object);
+                    LogConfiguration(p.Object);
 
-                        // retrieve application
-                        var application = container.GetInstance<XmlLookupApplication>();
+                    // configure DI container
```

Comment:
This comment seems to have been misplaced.

Reply:
done